sections/02_intro.md

```diff
@@ -23,15 +23,25 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
-These methods are powered by improvements to artificial neural networks, an idea
-that was first proposed in 1943 [@doi:10.1007/BF02478259] as a model for how our
-brains process information. Since then, interest in neural networks as
-computational models has waxed and waned over time. This history is interesting
-in its own right [@doi:10.1103/RevModPhys.34.135]. In recent years, attention
-has shifted back to neural networks as hardware improvements have allowed the
-techniques that have come to be known as deep learning to surpass other machine
-learning algorithms. `TODO: add what is/isn't deep learning to this paragraph?
-Revise paragraph to also fix lengthy closing sentence. Issue #421`
+Deep learning methods have some common features. First, they encode data via
```

Comment:
Part of the reason this is hard to write is that we're doing it without any figures.  I agree that layer isn't fully explained, but I struggle to explain it concisely with text alone.  We could consider leaving this and work on adding an introductory figure here during revision.